src/useFormState.ts

```diff
@@ -82,6 +83,7 @@ let pendingAutoSave = false;
  */
 export function useFormState<T, I>(opts: UseFormStateOpts<T, I>): ObjectState<T> {
   const { config, init, addRules, readOnly = false, loading, autoSave } = opts;
+  const autoSaveStatusContext = useAutoSaveStatus();
```

Comment:
Just curious, any reason to not destructure `{ triggerAutoSave, resolveAutoSave }`?

Reply:
It seemed a little ancillary so I wanted to keep it "bundled" together under `autoSaveStatusContext.[whatever]`.

If I was a few lines down and just saw `triggerAutoSave()` I might think it's more significant than it is. In this _particular_ case (not a general rule) ,`autoSaveStatusContext.triggerAutoSave()` communicates more accurate information to me than a stand-alone `triggerAutoSave()`.